main/main.cpp

```diff
@@ -276,6 +276,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("\n");
 
 	OS::get_singleton()->print("Standalone tools:\n");
+	OS::get_singleton()->print("  -i, --init <name> [<path>]       Create a new project.\n");
```

Comment:
This should be within `#ifdef TOOLS_ENABLED` (and same for the parsing of `-i` and `--init`).